sale_milestone_profile_invoicing/models/sale_order_line.py

```diff
@@ -149,7 +149,7 @@ def _compute_invoice_status(self):
         for line in self:
             if line._is_linked_to_milestone_product():
                 mapping = line.project_id.sale_line_employee_ids.filtered(
-                    lambda r: r.sale_line_id == line.id
```

Comment:
This should have been in a separated PR, I think

Reply:
I will keep them here to make travis green, for time economy, later it can be moved to separate PR

Reply:
moved to https://github.com/OCA/sale-workflow/pull/891